controllers/providers/pxc/applier.go

```diff
@@ -117,6 +120,7 @@ func (p *applier) Engine() error {
 		Advanced: common.DefaultAffinitySettings().DeepCopy(),
 	}
 	// We preserve the settings for existing DBs, otherwise restarts are seen when upgrading Everest.
+	// Additionally, we also need to check for the spec changes, otherwise the user can never voluntarily change the resource setting.
```

Comment:
You added this comment here but the spec change check was added above :sweat_smile:

Reply:
My bad. Left overs from some refactoring. Fixed 😅